Sources/SwiftProtobuf/Message.swift

```diff
@@ -115,11 +115,10 @@ public protocol Message: CustomDebugStringConvertible {
 }
 
 public extension Message {
-
-  /// If the generated code needs to provide its own implementation, usually
-  /// because the underlying `.proto` file uses proto2 syntax, it will provide
-  /// its own implementation. Generally, users of the generated code should
-  /// not.
+  /// Generated proto2 messages that contain required fields (or nested messages
```

Comment:
Extensions also come into play (if the message is extendable, then an extension could be added that is proto2)